game/utils/keycodec.cpp

```diff
@@ -57,10 +57,12 @@ std::initializer_list<KeyCodec::K_Key> KeyCodec::keys = {
   {Tempest::Event::K_LControl, 0x1d00},
   {Tempest::Event::K_LShift,   0x2a00},
   {Tempest::Event::K_LAlt,     0x3800},
+  {Tempest::Event::K_LCommand, 0x4800},
```

Comment:
How `0x4800` and `0x4801` were selected?
While I mention before, that can be anything really; curious if you can check what vanilla(Wineskin) writes in *.ini for those keys

Reply:
Wineskin has checkboxes for remapping these that I think I posted before:
<img width="167" alt="Screenshot 2023-02-06 at 21 59 34" src="https://user-images.githubusercontent.com/12349477/217084280-2d3720bf-0875-4bba-b271-1b5918f781ae.png">

With the checkboxes **unchecked**:
- left command is interpreted as left alt: `3800`
- right command is interpreted as right alt: `b800`
- left ctrl is interpreted as left ctrl: `1d00`
- my keyboard doesn't have a right ctrl, but I'd expect it to work the same as with left control (so right ctrl - `9d00`)
- option/alt keys do nothing

With the checkboxes **checked**:
- left command is interpreted as left ctrl: `1d00`
- right command is interpreted as right ctrl: `9d00`
- left ctrl is interpreted as left ctrl: `1d00`
- left alt is interpreted as left alt: `3800`
- right alt is interpreted as right alt: `b800`

So with the checkboxes checked, ctrl and command do the same thing. I don't think there is a "cannonical" code for either command keys. So I picked the  values at random really - tried to find something that wasn't occupied, but also wanted the two to be close to each other. If you prefer some other codes, feel free to edit them.